docs/components.md

```diff
@@ -21,8 +21,14 @@ validateheaders:
       - "pm"
 ```
 
+`validate_headers` also supports splitting on a configurable delimiter for each `allowed` header. To accomplish this, we set a `split` configuration using the headers name and the character(s) we want to `split` on.
```

Comment:
Do we allow multi-char?

Reply:
yep! we allow it. I can't think of a case where we not want to allow multi-char. YAGNI. And if this case arises in the future, we can iterate on this

Reply:
we do support multi-char splits at this time and wrote some tests to verify this behavior works